src/operator/subgraph/tensorrt/tensorrt-inl.h

```diff
@@ -180,6 +253,17 @@ class TensorrtProperty : public SubgraphProperty {
     n->attrs.name = "TensorRT" + std::to_string(subgraph_id);
     n->attrs.op = Op::Get("_TensorRT");
     CHECK(n->attrs.op);
+    // prevent using Gamma value if using fix_gamma on BatchNorm
+    DFSVisit(new_sym.outputs, [&n](const nnvm::NodePtr& node) {
+      if (node->op() == Op::Get("BatchNorm")) {
```

Comment:
Why not just check for FixGamma = true during nnvm -> Onnx conversion and set gamma to 0 if it's true?

Reply:
I think this one should be handled upstream.  Have a look at https://github.com/apache/incubator-mxnet/commit/6b22aa455e562db0e2744e94dc6a583191081b05 and let us know if you'd rather use this fix or if ours looks ok.

Reply:
I think solution 6b22aa4 looks better, I removed mine